### ISSUE TITLE
INFO and WARNING log output is written to STDERR

### ISSUE THREAD

**Issue by HeikoOnnebrink** [wal-g/wal-g#297]
When using wal-g inside some automation context after a successful execution we  would expect log and info output to be written to STDOUT. When some errors appear and exit code is != 0 we expect error message to be written to ERROUT. This is standard behaviour of apps.

WAL-G seems to write all log output to ERROUT not distinguishing between log types
https://github.com/wal-g/wal-g/blob/master/internal/tracelog/logging.go#L17..L20

This makes usage complicated. When exit code is != 0 I have not the isolated error in STDERR but its combined with all other (INFO and WARNING) messages.

Wouldn't it make sense to send INFO and WARNING to STDOUT and send just ERROR output to STDERR or do is misunderstand something here ?

I raised this topic on slack and @x4m suggested to create this issue and ask @Tinsane and @davidfetter to join the discussion to make a decision

**Comment by g0djan** [comment]
Fixed

**Comment by HeikoOnnebrink** [comment]
@g0djan I tested 0.2.14 and 0.2.14a releases .. but both seem not to include the fix.
is there any release that contains the fix or do I have to build it myself from master ?

**Comment by HeikoOnnebrink** [comment]
@g0djan short update ... I am running now since 3 days a custom build wal-g bin and all works perfect.. did not encounter any side-effects.. :-) 
from my side no concerns to include it into the next release build ..

**Comment by x4m** [comment]
So the problem is that tracelog is not bumped?

**Comment by Melkij** [comment]
@x4m , yes, most likely this is the only reason. go.mod still mention version without fix.

**Comment by Melkij** [comment]
@x4m , @g0djan , any updates here? go.mod still points to github.com/wal-g/tracelog v0.0.0-20190824100002-0ab2b054ff30 - the version prior to the fix.

**Comment by HeikoOnnebrink** [comment]
@x4m and @g0djan 
last time I got stuck here I rolled my own wal-g binary that I run in prod since then. I would like to switch back to the official released wal-g binaries and consume all changes applied since then, but I just recognised that https://github.com/wal-g/wal-g/blob/master/go.mod#L54 still points to the old version of tracelog.
Any chance to get it bumped up to the current version?
many thanx in advance 
cheers
Heiko
